docs/api/covidcast-signals/fb-survey.md

```diff
@@ -315,6 +317,23 @@ also available. These have names beginning `smoothed_w`, such as
 `smoothed_wtested_14d`.
 
 
+## Vaccination Indicators
+
+| Signal | Description | Survey Item |
+| --- | --- | --- |
+| `smoothed_accept_covid_vaccine` | Estimated percentage of respondents who would definitely or probably choose to get vaccinated, if a COVID-19 vaccine were offered to them today | V3 |
+
+This indicator is based on questions added in Wave 6 of the survey, introduced
+on December 19, 2020. **Note:** As of January 2021, vaccination items on the
+survey are being revised in preparation for Wave 7. The signal above may be
+replaced with a new signal (with different name) if the underlying survey item
+changes significantly.
```

Comment:
```suggestion
survey are being revised in preparation for Wave 7. We may replace the signal above with a new signal (with a different name) if the underlying survey item changes significantly.
```

I would use an active voice, but if it's not terribly important no worries. The 'a' before different should be kept though.